src/barkhaPJun20/FindMissingNumInArray.java

```diff
@@ -0,0 +1,21 @@
+package barkhaPJun20;
+
+public class FindMissingNumInArray {
+	int arraySum=0;
+	int originalSum=0;
+	
+	void printMissingNumInArray(int[] arrayNum) {
+		for (int index=0; index<arrayNum.length; index++) {
+			arraySum=arraySum+arrayNum[index];
```

Comment:
You can write for loop (for array values) inside for loop  (for range specified) i and compare the the two with each other.

Reply:
Wrote this method as well. In fact wrote both methods so in future I can refer both.Thank you.

Reply:
Code looks good to me. :)